prow/plugins/verify-owners/verify-owners.go

```diff
@@ -321,9 +321,9 @@ func handle(ghc githubClient, gc *git.Client, roc repoownersClient, log *logrus.
 
 		// prune old comments before adding a new one
 		cp.PruneComments(func(comment github.IssueComment) bool {
-			return strings.Contains(comment.Body, fmt.Sprintf(nonCollaboratorResponseFormat, ownersFileName, org))
+			return strings.Contains(comment.Body, fmt.Sprintf(nonCollaboratorResponseFormat, ownersFileName, org, triggerConfig.JoinOrgURL))
 		})
-		if err := ghc.CreateComment(org, repo, number, markdownFriendlyComment(org, nonTrustedUsers)); err != nil {
+		if err := ghc.CreateComment(org, repo, number, markdownFriendlyComment(org, triggerConfig.JoinOrgURL, nonTrustedUsers)); err != nil {
```

Comment:
Almost feels like we should construct this with the github URL configured and org/repo?

Reply:
Unless that link is a how-to, nevermind :man_facepalming: